doc/how_to_validate.rst

```diff
@@ -0,0 +1,35 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _tutorial::
+
+##########################################
+How to validate?
+##########################################
+
+Data workflow is PyPSA-Earth allows to extract all the data needed to build your energy model from global open data sources. However, data availability for many parts of the world is still quite limited and it's essential to validate the data included into the model. Usually the best sources to compare with are regional data hubs along with harmonized datasets of international organisations such as: 
```

Comment:
I'd start highlighting the sample notebook we have.
Then, I'd add a list of actions needed to do the validation. The list can be derived by the notebook itself

Reply:
Agree and will elaborate that

Reply:
Done